pandas/tests/extension/test_ops.py

```diff
@@ -0,0 +1,20 @@
+import operator
+from decimal import Decimal
+
+import pandas as pd
+import pandas.util.testing as tm
+from pandas.tests.extension.decimal_array import DecimalArray
+
+
+def test_combine_from_sequence_raises():
```

Comment:
why is this not with the existomg tests?

Reply:
This isn't a base extensionarray test that applies to all EAs. It's just to EAs that may raise errors other than TypeError in `_values_from_sequence`.

We need a place for simple tests like this that

1. Aren't applicable to every downstream ExtensionArray (so not in `tests/extension/base`)
2. Are testing pandas dispatching, rather than the behavior of a specific EA (so not in `tests/arrays/`)

This seemed like a sensible home.